### DISCUSSION TITLE
Best equipment for playing trombone in Jamulus

### DISCUSSION THREAD

**Discussion by uherbst** [jamulussoftware/jamulus#1389] [General]
Hi there,
I'm trying to improve my sound for playing trombone in Jamulus.

Current setup:
MacBook (2019) with ANC-headspeakers (cable). 
As microphone I use the built in mic.

Will sound be significantly better, if I use a audio device + analog microphone ?
Or is it useful to couple the mac with my Yamaha silent brass (with build in microphone) ?

Thank you for your tips
Uli

**Comment by chrisrimple** [comment]
You'll definitely get better sound if you purchase an audio interface (simple like Focusrite Scarlett Solo is enough) and a quality microphone.  I can't speak to Silent Brass (I'm not a horn player), but I'm aware that others are using it successfully with Jamulus.

**Reply by uherbst** [reply]
Any recommendation for a quality microphone ?

**Comment by henkdegroot** [comment]
Hi, I would not recommend using a silent brass system for this. The sound will not be as natural as the normal sound.
However, if you have to use it (i.e. because you don't want to distrub the neighbours when you're playing in the middle of the night) you can.
A cheap audio interface with a reasonable microphone will provide a much better result.

One of the persons in the video linked below is using the audio interface from Behringer (model UMC22). Not sure about the microphone he is using. I can check...if you want.
[Trombone Trio](https://youtu.be/adWgKnf2KYE)

For more good audio devices, you can check here: [Jamulus-Sound-Devices](https://jamulus.io/kb/2021/01/05/Jamulus-Sound-Devices.html)

**Comment by leoguiders** [comment]
I'm using an Audient Evo4 interface with an Audix i5 dynamic Mic for my trombone. 
The other trombone players in my group are using the Samson C01U PRO USB mic (condenser), which is working fine with ASIO4ALL drivers (experience on MacOS may vary).

Advantage of a condenser is that you don't have to move that close to the mic when you switch from playing to talking.

**Comment by pieterbos** [comment]
Yes, it will be better, even a €30 microphone will be better than the macbook. I do not like the silent brass sound at all, have not used it for years.

What's the budget?

A focusrite is fine as an audio interface. The Behringer UMC202HD is still decent with a mac, but the latency is not great on a mac - measured with my macbook at 10.5ms audio interface latency, on Windows about 7ms. in the whole setup, only a small difference, but still.

For the mic, I think an audio-technica at2020 sounds pretty good. If lower budget, a Behringer C-1 is not bad at all - a bit on the noisy side, and not great, and certainly not the best option if you also need it for bass trombone, but miles ahead of your macbook microphone.

I use a Nohype audio ribbon mic, but that's yet a different budget, and that certainly has its drawbacks as well.

**Comment by dcorson-ticino-com** [comment]
It depends on how much you prize good sound.
I started with a condenser instrument mic, but I found it noisy and started looking for better, but used to keep down the investment.
I am very happy now with a used Rode NT5, at 1/2 of new price great sound and no noise at all (almost).
A good mic makes a big difference, but is not a necessity.

**Comment by Andrew999999** [comment]
Not sure how you got on with this, but I upgraded from a cheap condenser mic to one with a larger diaphragm (30mm) and it made an enormous difference to quality of sound - but also reduced background noise significantly.  I also use Sony sound cancelling headphones which are great when I'm playing a live instrument (rather than keyboard) like trombone, as it cuts down direct transmission of sound to my ears - which can be distracting.

**Comment by rdica** [comment]
Oddly no one's mentioned putting a compressor in the chain, the one piece of gear that will help prevent clipping and save everyones elses ears in the process. Especially useful for condensers :)
